content/news/2020-12-01-about-me.md

```diff
@@ -0,0 +1,74 @@
+title: All about my Outreachy internship with Mixxx DJ: Intro
+authors: Aanyu Deborah Oduman
+tags: website, Outreachy, internhip
+date: 2020-12-01 23:15:00
+## A little bit about me.
+
+<table>
+  <tr>
+     <td></td>
+     <td></td>
+  </tr>
+  <tr>
+    <td><img src="/images/news/hello.jpg" width=450 height=360></td>
+    <td><img src="/images/news/deborah_intern.jpg" width=214 height=390></td>
+  </tr>
+ </table>
```

Comment:
Please try to avoid inline HTML.

In this case I suggest to just remove this part and use a different mechanism instead:

```suggestion
```

Add yourself to the `AUTHORS_METADATA` dict in the `pelicanconf.py` instead: https://github.com/mixxxdj/website/blob/website/pelicanconf.py#L158

Add a tagline (e.g. `Outreachy 2020 contributor`) and specify your github username. Then run `tools/update_avatars.py`. It will automatically fetch your profile picture from github and add it to the source tree (don't forget to `git add` it).

The author information including the avatar will be displayed below the post.

Reply:
> Please try to avoid inline HTML.
> 
> In this case I suggest to just remove this part and use a different mechanism instead:
> 
> Add yourself to the `AUTHORS_METADATA` dict in the `pelicanconf.py` instead: https://github.com/mixxxdj/website/blob/website/pelicanconf.py#L158
> 
> Add a tagline (e.g. `Outreachy 2020 contributor`) and specify your github username. Then run `tools/update_avatars.py`. It will automatically fetch your profile picture from github and add it to the source tree (don't forget to `git add` it).
> 
> Something like:
> 
> ```python
> AUTHOR_METADATA = {
>     # ...
>     "Aanyu Deborah Oduman": {
>         "github": "deborahtrez",
>         "tagline": "Outreachy contributor (Dec 2020 - Mar 2021)"
>     },
> }
> ```
> 
> The author information including the avatar will be displayed below the post.

Only problem is that one of the images comes out too big.  According to this markdown documentation, https://www.mkdocs.org/user-guide/styling-your-docs/#customizing-a-theme , i could create a css file in that directory and put the resizing css styles in there. I tried that but it didn't work. Other solutions  suggested on Stack work for some platforms and not for others (like Mixxx's). 
I'm going to comment it out in the mean time. :)

Reply:
> > Please try to avoid inline HTML.
> > In this case I suggest to just remove this part and use a different mechanism instead:
> > Add yourself to the `AUTHORS_METADATA` dict in the `pelicanconf.py` instead: https://github.com/mixxxdj/website/blob/website/pelicanconf.py#L158
> > Add a tagline (e.g. `Outreachy 2020 contributor`) and specify your github username. Then run `tools/update_avatars.py`. It will automatically fetch your profile picture from github and add it to the source tree (don't forget to `git add` it).
> > Something like:
> > ```python
> > AUTHOR_METADATA = {
> >     # ...
> >     "Aanyu Deborah Oduman": {
> >         "github": "deborahtrez",
> >         "tagline": "Outreachy contributor (Dec 2020 - Mar 2021)"
> >     },
> > }
> > ```
> > 
> > 
> > The author information including the avatar will be displayed below the post.
> 

 I found that i could use this syntax to achieve the same thing. `| [![hello]({static}/images/news/hello.jpg)]({static}/images/news/hello.jpg?) | [![deborah aanyu]({static}/images/news/deborah_intern.jpg)]({static}/images/news/deborah_intern.jpg) |
|:---:|:---:|`
Only problem is that one of the images comes out too big. According to this markdown documentation, https://www.mkdocs.org/user-guide/styling-your-docs/#customizing-a-theme , i could create a css file in that directory and put the resizing css styles in there. I tried that but it didn't work. Other solutions suggested on Stack work for some platforms and not for others (like Mixxx's).
> I'm going to comment it out in the mean time. :)

Reply:
> > > Please try to avoid inline HTML.
> > > In this case I suggest to just remove this part and use a different mechanism instead:
> > > Add yourself to the `AUTHORS_METADATA` dict in the `pelicanconf.py` instead: https://github.com/mixxxdj/website/blob/website/pelicanconf.py#L158
> > > Add a tagline (e.g. `Outreachy 2020 contributor`) and specify your github username. Then run `tools/update_avatars.py`. It will automatically fetch your profile picture from github and add it to the source tree (don't forget to `git add` it).
> > > Something like:
> > > ```python
> > > AUTHOR_METADATA = {
> > >     # ...
> > >     "Aanyu Deborah Oduman": {
> > >         "github": "deborahtrez",
> > >         "tagline": "Outreachy contributor (Dec 2020 - Mar 2021)"
> > >     },
> > > }
> > > ```
> > > 
> > > 
> > > The author information including the avatar will be displayed below the post.


I found that i could use this syntax to achieve the same thing. `| [![hello]({static}/images/news/hello.jpg)]({static}/images/news/hello.jpg?) | [![deborah aanyu]({static}/images/news/deborah_intern.jpg)]({static}/images/news/deborah_intern.jpg) | |:---:|:---:|`
 Only problem is that one of the images comes out too big. According to this markdown documentation, https://www.mkdocs.org/user-guide/styling-your-docs/#customizing-a-theme , i could create a css file in that directory and put the resizing css styles in there. I tried that but it didn't work. Other solutions suggested on Stack work for some platforms and not for others (like Mixxx's).

 I'm going to comment it out in the mean time. :)

Reply:
Please add yourself to the authors dictionary anyway.

Tables are semantically not really suitable for images. It will cause issues, especially on mobile:

![Screenshot_20201229_155813_com.duckduckgo.mobile.android.jpg](https://user-images.githubusercontent.com/1834516/103292764-c1f47680-49ee-11eb-9194-b930997c52f2.jpg)

If you want to display the two pictures side-by-side, you could just merge them into a single image using an image editor.

Or just remove them and use the author avatar mechanism instead.

Reply:
I decided to remove them